src/main/java/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,63 @@
+package main.java.duke;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task{
+    String taskType = "D";
+    LocalDate localDate = null;
+    LocalTime localTime = null;
+    String[] descSplitBySlash = null;
+    String[] descSplitBySpace = null;
+
+    public Deadline(String description) {
+        super(description);
+    }
+
+    public LocalTime stringToLocalTime(String hour, String minutes) {
+        LocalTime lc = LocalTime.parse(hour +":" + minutes);
+        return lc;
+    }
+
+    /**
+     * Returns a formatted description of the Deadline task, with the date and time dateline wrapped in parenthesis
+     * @return a formatted description of the Deadline task, with the date and time dateline wrapped in parenthesis
```

Comment:
Perhaps you can change the description for the return statement, as it does not give any additional information not present in your description!